string name
string status
---
bool success
